daemons/fenced/fenced_remote.c

```diff
@@ -95,7 +96,7 @@ static void
 free_remote_query(gpointer data)
 {
     if (data) {
-        st_query_result_t *query = data;
+        peer_device_info_t *query = data;
```

Comment:
seems to be the last place where other names weren't switched to peer - no need to change but ...

Reply:
might as well stay consistent